INSTALL.md

```diff
@@ -58,5 +59,5 @@ keen users, although contributions to this document are welcome.
 
     $ ./autogen.sh
     $ ./configure
-    $ make
-    $ sudo make install
+    $ gmake
+    $ sudo gmake install
```

Comment:
I like the original for its familiarity to system administrators. How about instead adding:

If the local host does not use GNU make by default (such as most BSD-based systems), use "./configure MAKE=gmake" instead.